### PR TITLE
docs: add pastcompany section to the pricing page

### PR DESCRIPTION
**Pull request by tianzhou**
CleanShot 2022-07-03 at 18 23 22

### CODE REVIEW

File: pages/pricing/index.vue [hunk 1]
```diff
@@ -17,6 +17,10 @@
         </h2>
       </div>
 
+      <div class="flex justify-center">
```
Comment (boojack) [inline pages/pricing/index.vue:20]: Looks like we should add a spacing, `pt-8 pb-4`?

Comment (tianzhou) [reply pages/pricing/index.vue]: Adjusted

CleanShot 2022-07-03 at 18 50 59

**Comment by tianzhou** [comment]
PTAL

**Review by boojack — APPROVED** [review]
LGTM

**Pull request merged**